enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/TopicImpl.java

```diff
@@ -143,6 +143,7 @@ public TopicModel getModel() {
 	@Override
 	public void setModel(TopicModel model) {
 		this.model = model;
+		this.modelId = model.getIdentifier();
```

Comment:
please look at the specs, the modelId must be a URI built using the endpoint base url,